sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2943,6 +2943,33 @@ class DataFrameSuite extends QueryTest
       .withSequenceColumn("default_index").collect().map(_.getLong(0))
     assert(ids.toSet === Range(0, 10).toSet)
   }
+
+  test("SPARK-35320 Reading JSON with key type different to String in a map should fail") {
```

Comment:
The convention is `SPARK-35320:`. Please, look test titles around.

Reply:
Fixed. Thanks